lib/auth/httpfallback.go

```diff
@@ -99,3 +100,23 @@ func (c *Client) generateHostCertHTTP(
 	}
 	return []byte(cert), nil
 }
+
+// TODO(Joerger): DELETE IN 16.0.0
```

Comment:
Leave a brief explanation after the delete message? Maybe "migrated to gRPC", like the others?

Reply:
I think it is self explanatory as it lives in `httpfallback.go`:
```go
// httpfallback.go holds endpoints that have been converted to gRPC
// but still need http fallback logic in the old client.
```